src/JupyterLibrary/clients/jupyterlab/Output.robot

```diff
@@ -39,3 +39,16 @@ Screenshot Each Output of Active JupyterLab Document
         Screenshot Each Output of Active JupyterLab Cell    ${prefix}_cell_${i}
         Screenshot Markdown of Active JupyterLab Cell    ${prefix}_cell_${i}
     END
+
+Wait Until a Given JupyterLab Code Cell Is Not Active In a Given Tab
+  [Documentation]  Waits until the given cell in a specific JL tab no longer has an active prompt "[*]:".
```

Comment:
This is... very verbose. Could this be broken down into multiple concepts? e.g. 
- get a handle to the `${cell}`
  - it might have just been added, or be selected by other means
- (maybe) go off and do other stuff
- `Wait Until JupyterLab Cell is Idle   ${cell}`

Also the terminology (idle, active, whatever) probably needs some investigation in either the DOM or message protocol.

Reply:
not sure I got what you mean by splitting into multiple concepts. i think it is useful to have a such high level keyword to use while running code cell.

I've just changed the keyword name to: `Wait Until JupyterLab Code Cell Is Idle`.